contrib/systemd/user_network.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=vsock_network
+After=NetworkManager.service
+
+[Service]
+ExecStart=/usr/libexec/podman/vm -preexisting -iface vsock0 -url vsock://2:%d/connect
+ExecStartPost=/usr/bin/nmcli c up vsock0
```

Comment:
I'm curious about "this may not be entirely necessary but wfm." in the commit log :) Better if we can avoid explicitly starting the network :)